src/main/java/persistence/sql/constant/BasicColumnType.java

```diff
@@ -0,0 +1,9 @@
+package persistence.sql.constant;
+
+public enum BasicColumnType {
```

Comment:
컬럼 타입을 별도로 만들어주셨네요! 어떠한 이유로 컬럼 타입을 만들어주셨을까요? 😄 

추가적으로 사소하지만 Basic~, Simple~ 등은 인터페이스의 기본 구현체에 주로 많이 사용하기 때문에 제거해주셔도 좋을 것 같아요!
```suggestion
public enum ColumnType {
```

Reply:
각 Column 클래스의 타입을 나타내기 위해서 작성했습니다. Java에서 제공되는 타입 혹은 단순 string 값보다 enum으로 만들어두면 필드가 나타내는 의미도 명확해지고 추후 타입 체크에 있어서도 이점이 있을거라고 생각했습니다.
하지만 지금 리뷰를 남기면서 생각해보니 결과적으로 쿼리를 만들때 해당 타입명으로 변환을 거쳐야한다는 점은 달라지지가 않아서 오히려 복잡도만 높인거 같다는 생각도 드네요..😭

이 점에 대해서 리뷰어님의 의견도 궁금합니다!

Basic 키워드에 대해서는 몰랐는데 수정하겠습니다! 꿀팁 감사합니다:)

Reply:
> ColumnType enum에 대한 리뷰어님의 의견도 궁금합니다!

이야기해주신 것처럼 상수로 사용하거나 타입 체크와 관련된 이점은 아쉽지만 지금 단계에서는 찾기 힘들다고 생각합니다! 🤔 
때문에 저는 이러한 경우 필요한 단계나 요구사항이 생겼을 때 클래스 분리를 고려해볼 것 같아요.

그럼 어떤 때에 ColumnType enum 클래스 사용을 고려하시나요? 하는 질문이 있으실 수 도 있을 것 같은데요!
저라면 ColumnType 을 공용으로 사용되는 JDBC의 컬럼의 타입으로 생각하고, 그에 따라 각 데이터베이스가 각각에 맞는 타입으로 다시 한번 변환하는 쪽으로 생각해서 사용을 고려할 것 같습니다! 😄 관련된 부분으로는 이번 주차 피드백에서 더욱 자세히 다룰 것 같은데요~! 혹시 추가적으로 질문이 있으시다면 편하게 DM 이나 코멘트 남겨주셔도 좋습니다 👍 👍